compass/landice/tests/humboldt/humboldt.cfg

```diff
@@ -22,8 +22,17 @@ low_log_speed = 0.75
 high_dist = 1.e5
 # distance within which cell spacing = min_spac (meters)
 low_dist = 1.e4
+# distance at which bed topography has no effect
+high_dist_bed = 1.e5
+# distance within which bed topography has maximum effect
+low_dist_bed = 5.e4
+# Bed elev beneath which cell spacing is minimized
+low_bed = 50.0
+# Bed elev above which cell spacing is minimized
```

Comment:
same as above - should this be "maximized"?

Reply:
```suggestion
# Bed elev above which cell spacing is maximized
```